crates/test/mock-consensus/src/keyring.rs

```diff
@@ -0,0 +1,24 @@
+pub struct Keys {
+    pub consensus_signing_key: ed25519_consensus::SigningKey,
+    pub consensus_verification_key: ed25519_consensus::VerificationKey,
+}
+
+impl Keys {
```

Comment:
We should really try to avoid creating something called just "keys", we have lots of different types of keys and it's important to be clear about what they are and what they do.

Reply:
e66628697 addresses this